lib/utils/tests-options-validator.js

```diff
@@ -48,7 +48,49 @@ Object.defineProperty(TestsOptionsValidator.prototype, 'shouldSplit', {
  */
 function validatePartitions(partitions, split) {
   validatePartitionSplit(partitions, split);
-  validatePartitionsUnique(partitions);
+  validateElementsUnique(partitions);
+}
+
+/**
+ * Validates the specified reply-browser
+ *
+ * @param {*} replayBrowser
+ * @param {*} replayExecution
+ */
+function validateReplyBrowser(replayBrowser, replayExecution) {
+  if(!replayExecution) {
+    throw new Error('You must specifiy a \'reply-execution\' value in order to use \'reply-browser\' option.');
```

Comment:
super nitpick here: 'path' instead of 'value' since reply-execution should be a path to the json file